model/github/github_repository.rb

```diff
@@ -17,6 +17,9 @@ class GithubRepository < Sequel::Model
   dataset_module Pagination
 
   CACHE_SIZE_LIMIT = 10 * 1024 * 1024 * 1024 # 10GB
+  def self.cache_size_limit
+    CACHE_SIZE_LIMIT
+  end
```

Comment:
Since we only use it here, is there any reason not to return it directly?

```suggestion
  def self.cache_size_limit
    10 * 1024 * 1024 * 1024 # 10GB
  end
```

Reply:
This calculates it every time instead of only once. Maybe with the JIT it ends up optimized the same, though. My preference would be to use `10_737_418_240 # 10GB` to avoid even the potential for an issue, though.